src/main/java/sim/coffee/OrderItem.java

```diff
@@ -1,5 +1,15 @@
 package sim.coffee;
 
+import java.math.BigDecimal;
+
 public abstract class OrderItem {
 
+    protected MenuItem menuItem;
+
+    public OrderItem(MenuItem m) {
+        menuItem = m;
+    }
```

Comment:
```suggestion
    public OrderItem(MenuItem menuItem) {
        this.menuItem = menuItem;
    }
```

Just to adhere to our UML. Also good practice to have self-documenting variables names even in such simple cases as this.